examples/SampleApp/src/components/NetworkDownIndicator.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { Spinner, useTheme } from 'stream-chat-react-native';
+
+
+const styles = StyleSheet.create({
+  networkDownContainer: {
+    alignItems: 'center',
+    flexDirection: 'row',
+  },
+  networkDownText: {
+    marginLeft: 10,
+  },
+});
+
+export const NetworkDownIndicator: React.FC<{ titleSize: 'small' | 'large' }> = ({ titleSize = 'small' }) => {
+    const {
+      theme: {
+        colors: { black },
+      },
+    } = useTheme();
+  
+    return (
+      <View style={styles.networkDownContainer}>
+        <Spinner />
+        <Text
+          style={[
+            styles.networkDownText,
+            {
+              color: black,
+            },
+            titleSize === 'large' ? {
+                fontSize: 16,
+                fontWeight: '700',
+            } : {}
```

Comment:
this should be within the StyleSheet too and then just used within the ternary. but this component is only ever used once it seems like and it's being used with the large titleSize so why not put that back within the networkDownText style like it was before?